lib/Twig/ExtensionSet.php

```diff
@@ -452,16 +454,28 @@ private function initExtension(Twig_ExtensionInterface $extension)
     {
         // filters
         foreach ($extension->getFilters() as $filter) {
+            if ($filter instanceof Twig_SimpleFilter) {
```

Comment:
Does this actually work, given that `Twig_SimpleFilter` is implemented as a class alias in 2.x ?